trace-sdk/src/main/java/io/bitrise/trace/TraceSdk.java

```diff
@@ -69,11 +74,21 @@ public synchronized static void init(@NonNull final Context context) {
             initDataCollection(context);
             initLifeCycleListener(context);
             initNetworkTracing();
+            TraceLog.i(String.format(LogMessageConstants.TRACE_DEBUG_FLAG_STATUS, DEBUG_ENABLED));
         } else {
             TraceLog.e(new TraceException.TraceConfigNotInitialisedException());
         }
     }
 
+    /**
+     * Flag to enable TraceSdk into debug mode.
+     * @param debugEnabled boolean value to enable or disable debug mode in the TraceSdk.
+     */
+    public synchronized static void setDebugEnabled(boolean debugEnabled) {
```

Comment:
nits: 
- please use the code formatter to add the missing extra line before param
- argument `debugEnabled` can be final

Reply:
Have updated - thank you.